miqa/learning/nn_training.py

```diff
@@ -399,7 +399,7 @@ def create_train_and_test_data_loaders(df, count_train):
     motion = CustomMotion(p=0.2, degrees=5.0, translation=5.0, num_transforms=1)
     inhomogeneity = CustomBiasField(p=0.1)
     spike = CustomSpike(p=0.1, num_spikes=(1, 1))
-    gamma = CustomGamma(p=0.1)  # after quick experimentation: gamma does not appear to help
+    # gamma = CustomGamma(p=0.1)  # after quick experimentation: gamma does not appear to help
```

Comment:
Did this somehow fix #344?

Reply:
It looks like the warnings are still there ([last test run](https://github.com/OpenImaging/miqa/runs/5322666382?check_suite_focus=true#:~:text=164-,miqa/core/tests/test_import.py%3A%3Atest_import_csv%5Bsuperuser%5D,scipy.ndimage%60%20namespace%2C%20the%20%60scipy.ndimage.morphology%60%20namespace%20is%20deprecated.,-169)). This just fixed the lint error that caused the tests to fail. The warnings don't cause a failure.